### ISSUE TITLE
Dashboard provisioning not reading new/changed dashboards

### ISSUE THREAD

**Issue by bradwood** [grafana/grafana#27714]
**What happened**:
I provisioned a grafana instance in k8s with prometheus. Datasources work, and stock dashboards work fine, but when I add an _additional_ dashboard provider/provisioner (`Beast` below) to a new Dashboard folder, json dashboard files are not read/recognised (although the folder itself is, thus showing only an empty dashboard folder).

![pic-selected-200922-2041-58](https://user-images.githubusercontent.com/3472045/93930019-9d647080-fd14-11ea-8a66-8de0a6b35954.png)

**What you expected to happen**:
I expect the dashboard JSON files to be periodically read into grafana and appear on the dashboard list in the appropriate folder. In this case, I included a single json dashboard in the `path` specified below which was create in the grafana UI itself( and loads and works fine when uploaded via the UI).

**How to reproduce it (as minimally and precisely as possible)**:
Dashboard provisioner config:
```
    apiVersion: 1
    providers:
    - name: "0"
      orgId: 1
      folder: Default
      type: file
      disableDeletion: true
      updateIntervalSeconds: 30
      allowUiUpdates: false
      options:
        path: /grafana-dashboard-definitions/0
    - name: "1"
      orgId: 1
      folder: Beast
      type: file
      disableDeletion: true
      updateIntervalSeconds: 30
      allowUiUpdates: false
      options:
        path: /grafana-dashboard-definitions/1
```
Grafani.ini:
```
[log]
mode = console
level = debug
```
Logs:
```
t=2020-09-22T17:01:02+0000 lvl=info msg="Starting Grafana" logger=server version=7.1.5 commit=9893b8c53d branch=HEAD compiled=2020-08-25T08:27:17+0000
t=2020-09-22T17:01:02+0000 lvl=info msg="Config loaded from" logger=settings file=/usr/share/grafana/conf/defaults.ini
t=2020-09-22T17:01:02+0000 lvl=info msg="Config loaded from" logger=settings file=/etc/grafana/grafana.ini
t=2020-09-22T17:01:02+0000 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.data=/var/lib/grafana"
t=2020-09-22T17:01:02+0000 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.logs=/var/log/grafana"
t=2020-09-22T17:01:02+0000 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.plugins=/var/lib/grafana/plugins"
t=2020-09-22T17:01:02+0000 lvl=info msg="Config overridden from command line" logger=settings arg="default.paths.provisioning=/etc/grafana/provisioning"
t=2020-09-22T17:01:02+0000 lvl=info msg="Config overridden from command line" logger=settings arg="default.log.mode=console"
t=2020-09-22T17:01:02+0000 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_PATHS_DATA=/var/lib/grafana"
t=2020-09-22T17:01:02+0000 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_PATHS_LOGS=/var/log/grafana"
t=2020-09-22T17:01:02+0000 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_PATHS_PLUGINS=/var/lib/grafana/plugins"
t=2020-09-22T17:01:02+0000 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_PATHS_PROVISIONING=/etc/grafana/provisioning"
t=2020-09-22T17:01:02+0000 lvl=info msg="Path Home" logger=settings path=/usr/share/grafana
t=2020-09-22T17:01:02+0000 lvl=info msg="Path Data" logger=settings path=/var/lib/grafana
t=2020-09-22T17:01:02+0000 lvl=info msg="Path Logs" logger=settings path=/var/log/grafana
t=2020-09-22T17:01:02+0000 lvl=info msg="Path Plugins" logger=settings path=/var/lib/grafana/plugins
t=2020-09-22T17:01:02+0000 lvl=info msg="Path Provisioning" logger=settings path=/etc/grafana/provisioning
t=2020-09-22T17:01:02+0000 lvl=info msg="App mode production" logger=settings
t=2020-09-22T17:01:02+0000 lvl=dbug msg="Initializing SqlStore" logger=server
t=2020-09-22T17:01:02+0000 lvl=info msg="Connecting to DB" logger=sqlstore dbtype=sqlite3
t=2020-09-22T17:01:02+0000 lvl=info msg="Starting DB migration" logger=migrator
t=2020-09-22T17:01:02+0000 lvl=info msg="Executing migration" logger=migrator id="create migration_log table"

<snip lots of SQL logs>

t=2020-09-22T17:01:09+0000 lvl=dbug msg="Executing sql migration" logger=migrator id="add unique index cache_data.cache_key" sql="CREATE UNIQUE INDEX `UQE_cache_data_cache_key` ON `cache_data` (`cache_key`);"
t=2020-09-22T17:01:10+0000 lvl=info msg="Created default admin" logger=sqlstore user=admin
t=2020-09-22T17:01:10+0000 lvl=dbug msg="Initializing HTTPServer" logger=server
t=2020-09-22T17:01:10+0000 lvl=dbug msg="Initializing RenderingService" logger=server
t=2020-09-22T17:01:10+0000 lvl=dbug msg="Initializing manager" logger=server
t=2020-09-22T17:01:10+0000 lvl=dbug msg="Initializing HooksService" logger=server
t=2020-09-22T17:01:10+0000 lvl=dbug msg="Initializing OSSLicensingService" logger=server
t=2020-09-22T17:01:10+0000 lvl=dbug msg="Initializing PluginManager" logger=server
t=2020-09-22T17:01:10+0000 lvl=info msg="Starting plugin search" logger=plugins
t=2020-09-22T17:01:10+0000 lvl=dbug msg="Scanning core plugin directory" logger=plugins dir=/usr/share/grafana/public/app/plugins
t=2020-09-22T17:01:11+0000 lvl=dbug msg="Scanning bundled plugins directory" logger=plugins dir=/usr/share/grafana/plugins-bundled
t=2020-09-22T17:01:11+0000 lvl=info msg="Registering plugin" logger=plugins name="Direct Input"
t=2020-09-22T17:01:11+0000 lvl=dbug msg="Scanning external plugins directory" logger=plugins dir=/var/lib/grafana/plugins
t=2020-09-22T17:01:11+0000 lvl=dbug msg="Getting signature state of plugin" logger=plugins plugin=input
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Verifying contents of plugin manifest" logger=plugins plugin=input
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing RemoteCache" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing InternalMetricsService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing AlertEngine" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing QuotaService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing ServerLockService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing UserAuthTokenService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing CleanUpService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing LoginService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing UsageStatsService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing SearchService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing TracingService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing NotificationService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing ProvisioningService" logger=server
t=2020-09-22T17:01:12+0000 lvl=info msg="inserting datasource from configuration " logger=provisioning.datasources name=prometheus uid=
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Looking for plugin provisioning files" logger=provisioning.plugins path=/etc/grafana/provisioning/plugins
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Validating plugins" logger=provisioning.plugins
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Looking for alert notification provisioning files" logger=provisioning.notifiers path=/etc/grafana/provisioning/notifiers
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Validating alert notifications" logger=provisioning.notifiers
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing DatasourceCacheService" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="NOTIFY_SOCKET environment variable empty or unset, can't send systemd notification" logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Waiting on services..." logger=server
t=2020-09-22T17:01:12+0000 lvl=dbug msg="No image renderer found/installed. For image rendering support please install the grafana-image-renderer plugin. Read more at https://grafana.com/docs/grafana/latest/administration/image_rendering/" logger=rendering
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Plugins: Adding route" logger=http.server route=/public/plugins/input dir=/usr/share/grafana/plugins-bundled/internal/input-datasource
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Found old rendered image to delete" logger=cleanup deleted=0 kept=0
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Looking for App Dashboard Updates" logger=plugins
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Initializing Stream Manager"
t=2020-09-22T17:01:12+0000 lvl=info msg="HTTP Server Listen" logger=http.server address=[::]:3000 protocol=http subUrl= socket=
t=2020-09-22T17:01:12+0000 lvl=dbug msg="starting cleanup of expired auth tokens" logger=auth createdBefore=2020-08-23T17:01:12+0000 rotatedBefore=2020-09-15T17:01:12+0000
t=2020-09-22T17:01:12+0000 lvl=dbug msg="cleanup of expired auth tokens done" logger=auth count=0
t=2020-09-22T17:01:12+0000 lvl=dbug msg="Checking for updates" logger=plugins
t=2020-09-22T17:01:13+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=0 action=Edit
t=2020-09-22T17:01:13+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:13+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/node-cluster-rsrc-use/node-cluster-rsrc-use.json folderId=1
t=2020-09-22T17:01:13+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:01:13+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:13+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:14+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:14+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/persistentvolumesusage/persistentvolumesusage.json folderId=1
t=2020-09-22T17:01:14+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:14+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:14+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:14+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/pod-total/pod-total.json folderId=1
t=2020-09-22T17:01:14+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:14+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:19+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:19+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/prometheus-remote-write/prometheus-remote-write.json folderId=1
t=2020-09-22T17:01:19+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:19+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:19+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:19+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/proxy/proxy.json folderId=1
t=2020-09-22T17:01:19+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:19+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:19+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:19+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/workload-total/workload-total.json folderId=1
t=2020-09-22T17:01:19+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:19+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:19+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:19+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/k8s-resources-workloads-namespace/k8s-resources-workloads-namespace.json folderId=1
t=2020-09-22T17:01:19+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:19+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/namespace-by-workload/namespace-by-workload.json folderId=1
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/k8s-resources-pod/k8s-resources-pod.json folderId=1
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/k8s-resources-workload/k8s-resources-workload.json folderId=1
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/node-rsrc-use/node-rsrc-use.json folderId=1
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/k8s-resources-cluster/k8s-resources-cluster.json folderId=1
t=2020-09-22T17:01:20+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:20+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/k8s-resources-node/k8s-resources-node.json folderId=1
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/namespace-by-pod/namespace-by-pod.json folderId=1
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/prometheus/prometheus.json folderId=1
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/statefulset/statefulset.json folderId=1
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/cluster-total/cluster-total.json folderId=1
t=2020-09-22T17:01:21+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:21+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:22+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:22+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/kubelet/kubelet.json folderId=1
t=2020-09-22T17:01:22+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:22+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:22+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:22+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/k8s-resources-namespace/k8s-resources-namespace.json folderId=1
t=2020-09-22T17:01:22+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:22+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:22+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:22+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/nodes/nodes.json folderId=1
t=2020-09-22T17:01:22+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:22+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:22+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:22+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/scheduler/scheduler.json folderId=1
t=2020-09-22T17:01:22+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:22+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:23+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:23+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/apiserver/apiserver.json folderId=1
t=2020-09-22T17:01:23+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:23+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:23+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:23+0000 lvl=dbug msg="saving new dashboard" logger=provisioning.dashboard type=file name=0 provisioner=0 file=/grafana-dashboard-definitions/0/controller-manager/controller-manager.json folderId=1
t=2020-09-22T17:01:23+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:23+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=1 action=Edit
t=2020-09-22T17:01:23+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T17:01:23+0000 lvl=dbug msg="User granted access to execute action" logger=dashboard.permissions userId=0 orgId=1 uname= dashId=0 action=Edit
t=2020-09-22T17:01:23+0000 lvl=dbug msg="Extracted alerts from dashboard" logger=alerting.extractor alertCount=0
t=2020-09-22T17:01:23+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:01:33+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:01:43+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:01:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T17:01:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T17:01:53+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:02:03+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:02:13+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:02:16+0000 lvl=eror msg="Failed to look up user based on cookie" logger=context error="user token not found"
t=2020-09-22T17:02:16+0000 lvl=eror msg="Failed to look up user based on cookie" logger=context error="user token not found"
t=2020-09-22T17:02:16+0000 lvl=eror msg="Failed to look up user based on cookie" logger=context error="user token not found"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/datasources/proxy/1/api/v1/query_range status=401 remote_addr=127.0.0.1 time_ms=1 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/annotations status=401 remote_addr=127.0.0.1 time_ms=1 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/datasources/proxy/1/api/v1/query_range status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:16+0000 lvl=eror msg="Failed to look up user based on cookie" logger=context error="user token not found"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/datasources/proxy/1/api/v1/query_range status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:16+0000 lvl=eror msg="Failed to look up user based on cookie" logger=context error="user token not found"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/datasources/proxy/1/api/v1/query_range status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/login/ping status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/login/ping status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/login/ping status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/login/ping status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:16+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/login/ping status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:20+0000 lvl=info msg="Successful Logout" logger=http.server User=
t=2020-09-22T17:02:20+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/logout status=302 remote_addr=127.0.0.1 time_ms=0 size=29 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:20+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/annotations status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:20+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/datasources/proxy/1/api/v1/query_range status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:20+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/datasources/proxy/1/api/v1/query_range status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:20+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/api/datasources/proxy/1/api/v1/query_range status=401 remote_addr=127.0.0.1 time_ms=0 size=26 referer="https://grafana.k8s.b5y.gs/d/728bf77cc1166d2f3133bf25846876cc/kubernetes-networking-workload?orgId=1&refresh=10s&var-datasource=prometheus&var-namespace=All&var-workload=node-exporter&var-type=daemonset&var-resolution=5m&var-interval=4h"
t=2020-09-22T17:02:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T17:02:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T17:02:23+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:02:26+0000 lvl=dbug msg="user auth token created" logger=auth tokenId=1 userId=1 clientIP=127.0.0.1 userAgent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36" authToken=cf7693aa19a23f36b1af082409ba46911b94dc344f81f9ccec21d5f1aee0f5c5
t=2020-09-22T17:02:26+0000 lvl=info msg="Successful Login" logger=http.server User=admin@localhost
t=2020-09-22T17:02:33+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:02:34+0000 lvl=dbug msg="seen token" logger=auth tokenId=1 userId=1 clientIP=127.0.0.1 userAgent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36" authToken=cf7693aa19a23f36b1af082409ba46911b94dc344f81f9ccec21d5f1aee0f5c5
t=2020-09-22T17:02:34+0000 lvl=dbug msg="Updating last user_seen_at" logger=context userId=1 orgId=1 uname=admin user_id=1
t=2020-09-22T17:02:34+0000 lvl=info msg="Request Completed" logger=context userId=1 orgId=1 uname=admin method=POST path=/api/user/password/reset status=400 remote_addr=127.0.0.1 time_ms=36 size=52 referer=https://grafana.k8s.b5y.gs/login
t=2020-09-22T17:02:37+0000 lvl=dbug msg="avatar.fetch(fetch new avatar): https://secure.gravatar.com/avatar/46d229b033af06a191ff2267bca9ae56?d=retro&r=pg&size=200"
t=2020-09-22T17:02:43+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:02:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T17:02:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T17:02:53+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:03:03+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:03:13+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:03:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T17:03:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T17:03:23+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:03:33+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:03:43+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:03:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T17:03:53+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:03:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T17:04:03+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:04:13+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:04:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T17:04:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T17:04:23+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:04:33+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:04:43+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T17:04:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0

<snip lots of repetition>


t=2020-09-22T19:31:42+0000 lvl=dbug msg="token needs rotation" logger=auth tokenId=1 authTokenSeen=true rotatedAt=2020-09-22T19:14:24+0000
t=2020-09-22T19:31:42+0000 lvl=dbug msg="auth token rotated" logger=auth affected=1 auth_token_id=1 userId=1
t=2020-09-22T19:31:42+0000 lvl=dbug msg="seen token" logger=auth tokenId=1 userId=1 clientIP=127.0.0.1 userAgent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36" authToken=5f47af8fe663f2d3875bf030808c43fb4f3fcb6b42a83b9872bfaa9bc483be21
t=2020-09-22T19:31:42+0000 lvl=dbug msg="Updating last user_seen_at" logger=context userId=1 orgId=1 uname=admin user_id=1
t=2020-09-22T19:31:43+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:31:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T19:31:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T19:31:53+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:32:03+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:32:13+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:32:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T19:32:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T19:32:23+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:32:33+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:32:43+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:32:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T19:32:53+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T19:32:53+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:33:03+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:33:13+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:33:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=0 path=/grafana-dashboard-definitions/0
t=2020-09-22T19:33:23+0000 lvl=dbug msg="Start walking disk" logger=provisioning.dashboard type=file name=1 path=/grafana-dashboard-definitions/1
t=2020-09-22T19:33:23+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0
t=2020-09-22T19:33:33+0000 lvl=dbug msg="Scheduling update" logger=alerting.scheduler ruleCount=0

```
**Anything else we need to know?**:
Stuff I've tried/checked:
* Just using the second provisioner only -- no difference
* checking all ownership and permissions on all files on disk to ensure that they are readable by the grafana process.
* various version of the grafana docker container (v7.1.0 upwards)

**Environment**:
- Grafana version: `v7.1.5 (9893b8c53d)`
- Data source type & version: Prometheus
- OS Grafana is installed on: Linux
- User OS & Browser: Linux + Chrome
- Grafana plugins: None

I've been scratching my head on this one for a whole evening to no avail, so any help would be much appreicated. From what I can tell from the log, the file simply does not appear to be seen by Grafana at all!

**Comment by marefr** [comment]
Have you restarted grafana in between changing the yaml file?

**Comment by bradwood** [comment]
No. I am relying on grafana to pick the dashboard up dynamically. Also, as this is running inside a kubernetes container, there is no init system, so I'm not entirely sure how to do that safely. I will try it though.. and report back...

**Comment by dprokop** [comment]
@marefr seems like this one is related to this feature request: https://github.com/grafana/grafana/issues/12878

**Comment by bradwood** [comment]
Just to be clear, @dprokop  I don't think this is related to `ConfigMaps` or the `kiwigrid/k3s-sidecar` which is used to inject files into the container via ConfigMap. In my scenario, those parts work. I have tried to _manually_ create JSON dashboards in the containers filesystem and these are not picked up either. This definitely feels like something specific to how the dashboard provisioner works, rather than any k8s background stuff.

**Comment by marefr** [comment]
Duplicate of #12878

**Comment by marefr** [comment]
If you want to do changes to the yaml config files and Grafana to puck them up please restart Grafana or use these HTTP API endpoints to trigger a reload: https://grafana.com/docs/grafana/latest/http_api/admin/#reload-provisioning-configurations

**Comment by bradwood** [comment]
@marefr your suggestion didn't work, so not sure why you just closed this issue. 

```
 http --auth admin POST https://grafana/api/admin/provisioning/dashboards/reload
http: password for admin@grafana
HTTP/1.1 200 OK
Cache-Control: no-cache
Connection: keep-alive
Content-Length: 40
Content-Type: application/json
Date: Fri, 25 Sep 2020 22:31:06 GMT
Expires: -1
Pragma: no-cache
Server: nginx/1.19.1
Strict-Transport-Security: max-age=15724800; includeSubDomains
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-Xss-Protection: 1; mode=block

{
    "message": "Dashboards config reloaded"
}
```

Please can you re-open this.

**Comment by bradwood** [comment]
I also tried POST `https://grafana/api/admin/provisioning/datasources/reload` which reloaded the datasource, but still the problem persists.

**Comment by wonderg** [comment]
Same 
curl -u admin:admin -X POST http://localhost:3000/api/admin/provisioning/datasources/reload
{"message":"Datasources config reloaded"}
but nothing got reloaded actually
